web_responsive/static/src/js/kanban_renderer_mobile.js

```diff
@@ -345,6 +363,7 @@
          *   and displayed
          */
         _moveToGroup: function (moveToIndex, animate) {
+            if (this.widgets.length == 0) return Promise.resolve();
```

Comment:
Please use "strict equality" operator. 
```suggestion
            if (this.widgets.length === 0)  {
                return Promise.resolve();
            }
```

Reply:
Done